bears/c_languages/ArtisticStyleBear.py

```diff
@@ -0,0 +1,865 @@
+                         allow_indent_classes: bool=True,
+                         allow_indent_modifiers: bool=True,
+                         allow_indent_switches: bool=True,
+                         allow_indent_cases: bool=True,
+                         allow_indent_namespaces: bool=False,
+                         allow_indent_labels: bool=True,
+                         allow_indent_preproc_block: bool=True,
+                         allow_indent_preproc_definition: bool=True,
+                         allow_indent_preproc_conditionals: bool=True,
+                         allow_indent_column_one_comments: bool=True,
+                         allow_pad_header_blocks: bool=True,
+                         allow_pad_operators: bool=True,
+                         allow_pad_parenthesis: bool=False,
+                         allow_pad_parenthesis_out: bool=False,
+                         allow_pad_parenthesis_in: bool=False,
+                         delete_empty_lines_in_func: bool=False,
+                         break_closing_braces: bool=False,
+                         break_elseifs: bool=False,
+                         break_one_line_headers: bool=False,
+                         add_braces_to_one_line_conditionals: bool=False,
+                         remove_braces_from_one_line_conditionals: bool=False,
+                         remove_comment_prefix: bool=True):
+        """
+        :param bracket_style:
+            Brace Style options define the brace style to use.
+            For example: Allman style uses braces that are broken from the
+            previous block. If set to ``allman``, prefer::
+
+                int Foo(bool isBar)
+                {
+                    if (isBar)
+                    {
+                        bar();
+                        return 1;
+                    }
+                    else
+                        return 0;
+                }
+
+            For example: Java style uses braces that are attached to the end of
+            the last line of the previous block. If set to ``java``, prefer::
+
+                int Foo(bool isBar) {
+                    if (isBar) {
+                        bar();
+                        return 1;
+                    } else
+                        return 0;
+                }
+
+            For example: Kernighan & Ritchie style uses linux braces. Opening
+            braces are broken from namespaces, classes, and function
+            definitions. The braces are attached to everything else, including
+            arrays, structs, enums, and statements within a function.
+            If set to ``kr``, prefer::
+
+                int Foo(bool isBar)
+                {
+                    if (isBar) {
+                        bar();
+                        return 1;
+                    } else
+                        return 0;
+                }
+
+            For example: Stroustrup style uses linux braces with closing
```

Comment:
We've used this convention in the `StylintBear` so I've used that style for consistency.